src/Mvc/Mvc.Testing/src/WebApplicationFactoryContentRootAttribute.cs

```diff
@@ -33,7 +33,7 @@ public sealed class WebApplicationFactoryContentRootAttribute : Attribute
         /// </param>
         /// <param name="contentRootPath">The path to the content root. This path can be either relative or absolute.
         /// In case the path is relative, the path will be combined with
-        /// <code><see cref="Directory.GetCurrentDirectory()"/></code></param>
+        /// <c><see cref="Directory.GetCurrentDirectory()"/></c></param>
```

Comment:
```suggestion
        /// <see cref="Directory.GetCurrentDirectory()"/></param>
```

This shouldn't have had `<code>` tags in the first place. We don't need to wrap a `<see>` link in `<code>` or `<c>`.